superset/assets/src/components/Menu/MenuObject.jsx

```diff
@@ -51,20 +51,16 @@ export default function MenuObject({ label, icon, childs, url, index }) {
       eventKey={index}
       title={navTitle}
     >
-      {childs.map((child, index1) =>
-        child === '-' ? (
-          <MenuItem key={`$${index1}`} divider />
-        ) : (
-          <MenuItem
-            key={`${child.label}`}
-            href={child.url}
-            eventKey={parseFloat(`${index}.${index1}`)}
-          >
-            <i className={`fa ${child.icon}`} />
-            &nbsp; {child.label}
-          </MenuItem>
-        ),
-      )}
+      {childs.map((child, index1) => (
```

Comment:
This is probably unnecessary since you're already removing them in FAB. The menu data comes from FAB so we may want to keep this logic in case we want to add separators later 🤷‍♂. I don't feel strongly about this though, since we'll probably stop using FAB for the menu data before long

Reply:
Agreed, this should be unnecessary

Reply:
Fair 'nuff. I reverted that bit.